JPSImagePickerController/JPSImagePickerController.m

```diff
@@ -61,6 +61,53 @@ - (void)viewDidLoad {
     [self addCancelButton];
     [self addFlashButton];
     [self addCameraSwitchButton];
+
+
+
+    self.motionManager = [[CMMotionManager alloc] init];
```

Comment:
This should probably be in `viewDidAppear:` to match the `stopAccelerometerUpdates` call in `viewWillDisappear`.